#iou meaage 
float64 IouScore
float64 Intersection
float64 Union
